lib/core/src/lib.rs

```diff
@@ -27,10 +27,81 @@ pub mod system {
 
 use display::world::World;
 use wasm_bindgen::prelude::*;
+use basegl_system_web::keyboard_engine::KeyboardEngine;
+use basegl_system_web::{window};
+use web_sys::KeyEvent;
 
 #[wasm_bindgen(start)]
 pub fn start() {
     let world = World::new();
     world.add_workspace("canvas");
     world.start();
+
+    init_keyboard_engine();
+}
+
+/// Init keyboard handlers on Window with the specific combinations
+///
+/// Combinations:
+///  - a
+///  - Shift-a
+///  - Control-Shift-a
+///  - Control-Space
+///  - Alt-Space
+fn init_keyboard_engine() {
+    let keyboard_engine = KeyboardEngine::new(&window().unwrap());
+    keyboard_engine.capture(
+        vec![
+            KeyEvent::DOM_VK_A
+        ],
+        Box::new(|| {
+            web_sys::console::log_1(&"Key pressed: a".into());
+        })
```

Comment:
A thing here - If you use `Box::new` explicitly it is leaking implementation detail. What if some day you came to an idea to use `Rc` instead of `Box`? All clients code will need to be changed, which is bad. The `capture` function should get a generic type and pack it to `Box` in smart constructor.

Reply:
I'm using `Box` here for allocating in the heap. I think that if we would like to use `Rc` for example then the value should be cloned from there and dereferenced.  Do you suggest to create something like?
```
fn capture<F: FnMut()> (..:, callback: F) { 
```